README.md

```diff
@@ -20,10 +20,9 @@
 ## ✏️ Design
 
 ### 📏 Entity-Relationship Diagram
-Waiting for the ERD to be uploaded.
+![ERD Diagram](https://github.com/Zhi-704/c11-apollo-bandcamp-tracker/blob/main/diagrams/ERD.png)
 ### 📐 Architecture Diagram
-Waiting for the AD to be uploaded.
-
+![Architecture Diagram](https://github.com/Zhi-704/c11-apollo-bandcamp-tracker/blob/main/diagrams/Architecture_Diagram.png)
```

Comment:
Double check you have done this correctly. When previewing the README, I can see the ERD but can't see the AD.

Reply:
It links to the main branch rather than the branch it's on. It should fix itself when merged

Reply:
This doesn't sound reassuring ...

Reply:
Don't worry trust me